src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverRobot = {};
```

Comment:
Не знаю, що таке inver :)
```suggestion
  const inversedRobot = {};
```